src/ray/raylet/scheduling/cluster_resource_scheduler.cc

```diff
@@ -246,16 +242,18 @@ int64_t ClusterResourceScheduler::GetBestSchedulableNode(
     return best_node;
   }
 
-  auto spread_threshold = spread_threshold_;
-  // If the scheduling decision is made by gcs, we ignore the spread threshold
-  if (actor_creation && RayConfig::instance().gcs_actor_scheduling_enabled()) {
-    spread_threshold = 1.0;
+  // If the local node is available, we should directly return it instead of
+  // going through the full hybrid policy since we don't want spillback.
```

Comment:
Discussed offline: to simplify, we can just reject the task immediately if it is not locally schedulable. This way, the policy will always be to queue at the home raylet until remote resources become available.

Reply:
Discussed offline: will keep the behavior as it is for now and discuss what the better behavior is later.